pkg/cmd/server/bootstrappolicy/web_console_role_test.go

```diff
@@ -65,6 +65,7 @@ var rolesToHide = sets.NewString(
 	"system:openshift:aggregate-to-admin",
 	"system:openshift:aggregate-to-edit",
 	"system:openshift:aggregate-to-view",
+	"system:openshift:aggregate-to-cluster-reader",
```

Comment:
@openshift/origin-web-console-committers I've added a new role here that I think should be hidden to the end user, (heads up per the test's comment)

Reply:
@mrogers950 @enj aha, the test lives!  But we can eliminate that test, might be a good time to do so?

Reply:
@benjaminapetersen I can do a follow-up to get rid of this test if it's no longer useful.

Reply:
@mrogers950 sounds good.  Happy to chat about it if that would be helpful.